tests/helpers/Stringer.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Behat\Mink\Tests\helpers;
+
+/**
+ * Class Stringer
```

Comment:
this short description is totally useless (your long description should become the short one)